exampleSite/content/docs/faq.md

```diff
@@ -12,7 +12,7 @@ menu:
     name: FAQ
 ---
 
```

Comment:
I use 120 as a maximum line length for Mainroad markdown content (exceptions: long links and code blocks, etc.). My fault. I need to mention this in the guidelines or set up automatic validation in the future.

I'm not sure what editing tools you use, so let me know if you can fix this on your end?

Reply:
Understood; I tried to go through and format for length.